as/thrift.js

```diff
@@ -45,6 +45,7 @@ function TChannelAsThrift(opts) {
         'must pass source as an argument');
 
     self.thriftSource = opts.source;
+    self.thriftFileName = opts.thriftFileName || 'service.thrift';
```

Comment:
Also please update https://github.com/uber/tchannel-node/blob/master/hyperbahn-client.js#L179-L197 to pass argument.